docs/trex_release-notes.md

```diff
@@ -10,9 +10,44 @@ layout: docs
 
 ---
 
-### Developer Preview  
+### Developer Preview (0.7.0)
+*November 29, 2017*
 
-Initial release of the Tableau Extensions API. Provides supports for dashboard extensions. 
+- Update of the Tableau Extensions API. 
+- Tableau Extensions API library: `tableau-extensions-0.7.0.js`
+- Tableau Desktop 10.5 RC, Tableau Server 10.5 (from [Extensions API Developer Preview](https://prerelease.tableau.com){:target="_blank"})
+
+**Breaking change** 
+
+- Schema change - Updated XSD file for the dashboard extensions manifest file (`.trex`). 
+  If you have an existing extension, you must update the `.trex` file to follow the new schema. There is a script you can run that converts the manifest file for you. Or you can manually make the changes. For more information about the manifest, see [Tableau Extension Manifest File]({{ site.baseurl }}\docs\trex_manifest.html). You can download the manifest conversion script from the **Extensions API Developer Preview** on [https://prerelease.tableau.com](https://prerelease.tableau.com){:target="_blank"}.
```

Comment:
maybe add a quick statement somewhere along the lines of "New constraints are added on certain fields"